plugins/georeplication/rest.go

```diff
@@ -171,11 +174,18 @@ func georepCreateHandler(w http.ResponseWriter, r *http.Request) {
 	// Workaround till {{ volume.id }} added to the marker options table
 	vol.Options["marker.volume-uuid"] = vol.ID.String()
 
+	//save volume information for transaction failure scenario
+	if err := txn.Ctx.Set("oldvolinfo", oldvolinfo); err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
```

Comment:
@Madhu-1 No logging required?

Reply:
added logging, In most of the cases we are not logging if txn.Set fails. I will improve logging in upcoming PR's